hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -202,6 +207,54 @@ spec = parallel $ do
     propBelowSizeLimit maxTxSize forAllFanout
     propIsValid forAllFanout
 
+  describe "acceptance" $ do
+    it "can close & fanout every collected head" $ do
+      prop_canCloseFanoutEveryCollect
+
+-- * Properties
+
+prop_canCloseFanoutEveryCollect :: Property
+prop_canCloseFanoutEveryCollect = monadicST $ do
+  let maxParties = 20
+  ctx@HydraContext{ctxContestationPeriod} <- pick $ genHydraContext maxParties
+  cctx <- pick $ pickChainContext ctx
```

Comment:
```suggestion
  chain_ctx <- pick $ pickChainContext ctx
```

Reply:
We don't use snake case. `chainCtx` would be an alternative here, but then I should also change it in all its `64` usages.